### PR TITLE
Update PHP and Shell.

### PR DESCRIPTION
**Pull request by 52fhy**

### CODE REVIEW

File: free-programming-books-zh.md [hunk 1]
```diff
@@ -650,11 +651,11 @@
 
 ### Shell
 
+* [Shell脚本学习系列教程](https://github.com/52fhy/shell-book)
```
Comment (eshellman) [inline free-programming-books-zh.md]: why not http://me.52fhy.com/shell-book/ ?

File: free-programming-books-zh.md [hunk 2]
```diff
@@ -598,6 +598,7 @@
 
 ### Python
 
+* [Python教程 - 廖雪峰的官方网站](http://www.liaoxuefeng.com/wiki/0014316089557264a6b348958f449949df42a6d3a2e542c000)
```
Comment (eshellman) [inline free-programming-books-zh.md]: Great book! need to put in alphabetical order- check the Travis CI output.

Comment (52fhy) [inline free-programming-books-zh.md]: Thank you!

File: free-programming-books-zh.md (hunk 1 above)
Comment (52fhy) [inline free-programming-books-zh.md]: Good idea!

**Pull request merged**